README.md

```diff
@@ -1,21 +1,416 @@
 # Secure FHIR Gateway and Proxy
 
-## What is FHIR Proxy?
-FHIR Proxy is a    gatekeeping server application that works in conjunction with Azure API for FHIR/FHIR Server for Azure (FHIR Server). When FHIR Proxy is deployed alongside FHIR Server in the same Azure Tenant, the FHIR Proxy becomes the public endpoint for the FHIR Server – standing in front of the FHIR Server’s API and acting as a checkpoint in the exchange of FHIR data to/from the FHIR Server.
+Secure FHIR Gateway and Proxy is an Azure Function based solution that:
+ + Acts as an intelligent and secure gateway to FHIR Servers
+ + Allows multi-tenant access and purpose driven security policies specialized access to a common FHIR Server
+ + Provides a consolidated approach to pre and post processing of FHIR Server Calls to support various access and result filtering or actions.</br>
+ + Is Integrated with Azure Active Directory for authentication and to provide Role based access control.</br>
+ + Acts as a FHIR specific reverse proxy rewriting responses and brokering requests to FHIR Servers</br>
+## Authentication and RBAC Authorization
+By default the proxy will configure and use Azure Active Directory (Azure AD) as an authentication provider.  You will also need to assign users/groups and/or service principals into specific server access roles in order to access the FHIR Server via the proxy.  You can also offload this responsibility to [API Management](https://azure.microsoft.com/en-us/services/api-management/)
 
-## What is the purpose of FHIR Proxy?
-FHIR Proxy gives FHIR Server administrators an extra set of controls for filtering, transforming, and/or redistributing information as it passes into and out of the FHIR Server. This enhanced control in the transfer of FHIR data is accomplished via FHIR Proxy’s services in three key areas:
-•	Fine-grained Role-Based Access Control
-•	SMART on FHIR App Scoping
-•	Triggerable Event Publishing
+## Pre and Post Processing Support
```

Comment:
Do we want to include a mention about the sample pre/post-processing module to use for creating your own module?